tests/Feature/Http/Controllers/Admin/SpaceControllerTest.php

```diff
@@ -26,7 +26,7 @@ protected function setUp(): void
     public function testCreateFormIsShown(): void
     {
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar `$authorizedUser` .



```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````